src/DataMiner/SLManagedAutomation/UIBuilder.cs

```diff
@@ -249,11 +249,11 @@ public class UIBuilder
 		/// <returns>This <see cref="UIBuilder"/> instance.</returns>
 		/// <example>
 		/// <code>
-		/// UIBuilder uibDialogBox1 = new UIBuilder();
+		/// UIBuilder uiBuilder = new UIBuilder();
 		/// // ...
 		/// UIBlockDefinition blockTextBox = new UIBlockDefinition();
 		/// // ...
-		/// uibDialogBox1.AppendBlock(blockTextBox);
+		/// uiBuilder.AppendBlock(blockTextBox);
```

Comment:
We should probably use the name uibDialogBox1 or something starting with uib. According the the docs the intellisense in cube is only triggered if the name starts with uib. I checked it in cube and uiBuilder does not trigger the intellisense

Reply:
You're right, I'll change them to uibDialogBox.